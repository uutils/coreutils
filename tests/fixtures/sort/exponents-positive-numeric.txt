10000K78
10E


1000EDKLD


100E6

50e10
+100000

