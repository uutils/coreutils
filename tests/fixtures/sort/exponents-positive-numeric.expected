





+100000
10E
50e10
100E6
1000EDKLD
10000K78
